src/sql/strong-oracle/create.sql

```diff
@@ -167,6 +173,40 @@ ALTER TABLE "L_TRANSFERS" ADD (CONSTRAINT "FK_STATUS_ID__TRANSFERS"
   ("STATUS_ID") ON DELETE SET NULL);
 /
 
+CREATE TABLE "L_TRANSFER_ADJUSTMENTS"
+(
+  "TRANSFER_ADJUSTMENT_ID" INTEGER NOT NULL,
+  "TRANSFER_ID"            VARCHAR(36) NULL,
+  "ACCOUNT_ID"             INTEGER NOT NULL,
+  "DEBIT_CREDIT"           VARCHAR2(10) NOT NULL,
+  "AMOUNT"                 NUMBER(32,16) DEFAULT 0 NULL,
+  "IS_AUTHORIZED"          SMALLINT DEFAULT 0 NOT NULL,
+  "MEMO"                   VARCHAR2(4000) NULL
+)
+/
+
+CREATE INDEX L_XPK_TRANSFER_ADJUSTMENTS ON "L_TRANSFER_ADJUSTMENTS"
+  ("TRANSFER_ADJUSTMENT_ID" ASC);
+
+ALTER TABLE "L_TRANSFER_ADJUSTMENTS" ADD CONSTRAINT PK_TRANSFER_ADJUSTMENTS
+  PRIMARY KEY ("TRANSFER_ADJUSTMENT_ID");
+
+CREATE UNIQUE INDEX L_XAK_TRANSFER_ADJUSTMENTS ON "L_TRANSFER_ADJUSTMENTS"
+  ("TRANSFER_ID" ASC, "ACCOUNT_ID" ASC);
+
+ALTER TABLE "L_TRANSFER_ADJUSTMENTS" ADD CONSTRAINT L_XAK_TRANSFER_ADJUSTMENTS
```

Comment:
It turns out that both lines are necessary because a unique index and a unique constraint are separate concepts in Oracle: http://stackoverflow.com/questions/7521817/oracle-unique-constraint-and-unique-index-question